datafusion-examples/examples/advanced_parquet_index.rs

```diff
@@ -211,7 +211,7 @@ async fn main() -> Result<()> {
     //
     // Note: in order to prune pages, the Page Index must be loaded and the
     // ParquetExec will load it on demand if not present. To avoid a second IO
-    // during query, this example loaded the Page Index pre-emptively by setting
+    // during query, this example loaded the Page Index pre-emptily by setting
```

Comment:
I think this is actually meant to be something different: 

```suggestion
    // during query, this example loaded the Page Index preemptively by setting
```

Reply:
Yeah, I got it, I will recover it